new-frontiers-14/frontier-station-14
# Apothecary directional Fan update and housekeeping.

This PR primarily removes the microwave from the Apothecary as Erhard has provided us with a technical alternative. It's time to remove the kitchen tools.

It also includes some housekeeping and feedback response changes.




## About the PR



- add: 
One new scrubber and vent to the main bay. Bringing the total to 2x scrubbers and 2x vents in the med bay.
Filing cabinet.
Cockpit window shutters and companion button.
New chemistry wall shelf and secure wall shelf to the medbay
New o2/n2 emergency locker to the docking airlock
New medical eva suit to the docking airlock

- remove: 
Microwave
small mapped beaker
engineering atmo pump
second drain by front desk

- tweak: 
Swap plasma for uranium power.
Swap plasma fuel locker for uranium fuel locker.
Adjusted piping for cleaner layout and less odd clutter. 
Adjusted various decals for consistency. Ditto under-door floor tiles.
Swapped out maintenance locker for Erhard's new plasma fuel locker. 
Swapped out custom fill toolbox for maintenance belt. Ideal for paramedic work really.
Swapped out standing paramedic wall lockers for Erhard's new suit lockers. Looks great and people were just throwing the old lockers into space.
Tweaked placement of the chem lab drain.
Changed engineering and cargo bay lights from sodium to standard
Default power level of the generator is now 16kw
Placement of various bits and bobs, such as lab coats and pens. 

## Why / Balance


Time to remove the kitchen tools.

## How to test


Spawn Apothecary.
- Try to microwave something aboard the microwave. There should not be a microwave. Eat cold burrito.
- Check vents and scrubbers. Check air alarm.
- Check APC power.
- Check generator power. Should be at 16kw.
- Test cockpit window shutters. Button in cockpit.

## Media


![ss+(2024-09-12+at+05 53 26)](https://github.com/user-attachments/assets/a4fb879f-01cb-4e8e-8eb1-bf7afdcb8ba5)

## Breaking changes


**Changelog**



🆑 
- remove: Apothecary microwave
- tweak: Tiny fans swapped to directional fans. Now runs on Uranium.
